### PR TITLE
Removes storybook package and related code

### LINKED ISSUE #1171: [Proposal] Remove storybook
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by tmarkley**
### Description
Removes the broken and unfinished storybook package and related code.

### Issues Resolved
Resolves #1171
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
    - [x] `yarn test:jest`
    - [x] `yarn test:jest_integration`
    - [x] `yarn test:ftr`
- [x] New functionality has been documented.
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by mihirsoni** [comment]
@tmarkley  I remember this had couple components playground, are we sure we are not in need of these playground or components any more ?

**Comment by tmarkley** [comment]
> @tmarkley I remember this had couple components playground, are we sure we are not in need of these playground or components any more ?

Yeah I think it could be worthwhile pursuing alternatives for a similar experience. I do think these playgrounds are nice for devs. The problem is that storybook has been broken since we forked so it's not serving any purpose at this point and it'd take a bit of effort to get it working again. Even then, the storybook dependencies are tied to multiple CVEs and their updates aren't always stable. See #1056 for an example.

**Review by seraphjiang — DISMISSED** [review]

**Comment by seraphjiang** [comment]
Support to clean up these unused storybooks.

**Comment by YeyunLU** [comment]
Will this change be added to version 1.3.0?

**Comment by tmarkley** [comment]
@YeyunLU if there is not pushback from the community in the proposal (#1171) then yes, I think we can target this for the 1.3.0 release. Do you have any concerns?

**Comment by kavilla** [comment]
> @YeyunLU if there is not pushback from the community in the proposal (#1171) then yes, I think we can target this for the 1.3.0 release. Do you have any concerns?

I think it seems good to merge to 1.3 since it doesn't work in 1.2 so it wouldn't be a breaking change.

**Comment by YeyunLU** [comment]
Thanks @tmarkley and @kavilla , looking forward to version 1.3.0.

File: src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class ValidatedDualRange extends Component<Props> {
   // @ts-ignore state populated by getDerivedStateFromProps
   state: State = {};
 
-  _onChange = (value: Value) => {
```
Comment (kavilla) [inline src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx:91]: I'm not able to change input control here:
```
http://localhost:5601//app/visualize#/create?type=input_control_vis&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(filters:!(),linked:!f,query:(language:kuery,query:''),uiState:(),vis:(aggs:!(),params:(controls:!((fieldName:hour_of_day,id:'1644025754232',indexPattern:d3d7af60-4c81-11e8-b3d7-01146121b73d,label:'',options:(decimalPlaces:0,step:1),parent:'',type:range)),pinFilters:!f,updateFiltersOnChange:!f,useTimeFilter:!f),title:'',type:input_control_vis))
```

**Review by kavilla — CHANGES_REQUESTED** [review]
Input control is breaking for me, which I believe is what is effecting the functional tests from succeeding.

File: src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx (hunk 1 above)
Comment (tmarkley) [reply src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx:91]: Yeah not sure why this was made private. Updating that now.

Comment (tmarkley) [reply src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx:91]: Turns out all of this was caused by discrepancies with the `@types/react` package in the yarn.lock file. I resolved that and now this change is not necessary.

**Review by boktorbb-amzn — APPROVED** [review]

**Comment by kavilla** [comment]
LGTM! My only nitpick is the using `git log` doesn't come with the functionality that github does so then in the message all I see is:
```
Removes storybook package and related code
    
Resolves #1171
```

Which doesn't really seem inclusive, especially in forks, we could probably add more details in why this was removed in the git message?

**Review by ananzh — APPROVED** [review]

**Comment by tmarkley** [comment]
> LGTM! My only nitpick is the using `git log` doesn't come with the functionality that github does so then in the message all I see is:
> 
> ```
> Removes storybook package and related code
>     
> Resolves #1171
> ```
> 
> Which doesn't really seem inclusive, especially in forks, we could probably add more details in why this was removed in the git message?

When I squash the commit it will include the PR number too, but yeah I'll add more details from the issue in the commit message.

**Comment by YeyunLU** [comment]
Hi @tmarkley, it seems code changes in this PR is not included in the latest version (1.3.1). May I know which version of 1.3 will include it? Thanks!

**Comment by kavilla** [comment]
Hello @YeyunLU, you are right. It would appear we failed to backport this issue. It will be there for `2.0`. A new minor version of `1.x` is unlikely relatively soon.

Apologies.

**Comment by jordanoliver23** [comment]
@kavilla , @tmarkley can you tell me what happened here? Did we remove the license? No?

**Pull request merged**